tspi/context.go

```diff
@@ -132,3 +132,11 @@ func (context *Context) CreateHash(hash crypto.Hash) (*Hash, error) {
 		hashAlg: hash,
 	}, nil
 }
+
+// GetCapability reads the requested capability and subcapability from the TPM.
```

Comment:
Can you expand this comment with some examples or point to the docs for how you'd use this? :)

Reply:
Added information on where to look for valid capabilities as well as their usage.